src/main/java/com/rakovets/course/java/core/practice/oop_classes_and_objects/Point.java

```diff
@@ -0,0 +1,31 @@
+package com.rakovets.course.java.core.practice.oop_classes_and_objects;
+
+public class Point {
+    private int x = 0;
+    private int y = 0;
+
+    public Point(int x, int y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    public int getX() {
+        return this.x;
+    }
+
+    public void setX(int x) {
+        this.x = x;
+    }
+
+    public int getY() {
+        return this.y;
+    }
+
+    public void setY(int y) {
+        this.y = y;
+    }
```

Comment:
"Violation: Getters and setters should be placed at the very end
Solution: Move getters and setters to the end"